### ISSUE TITLE
Integration test breaking on parsing response

### ISSUE THREAD

**Issue by alex-hancock** [DataBiosphere/azul#462]
```
======================================================================
ERROR: test_smartseq2_run (tests.integration.test_end_to_end_dcp.TestSmartSeq2Run)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/HumanCellAtlas/dcp/tests/integration/test_end_to_end_dcp.py", line 103, in test_smartseq2_run
    runner = self.ingest_store_and_analyze_dataset(dataset_fixture='Smart-seq2')
  File "/HumanCellAtlas/dcp/tests/integration/test_end_to_end_dcp.py", line 26, in ingest_store_and_analyze_dataset
    runner.run(dataset, run_name_prefix="integration")
  File "/HumanCellAtlas/dcp/tests/dataset_runner.py", line 58, in run
    self.assert_data_browser_bundles(run_name)
  File "/HumanCellAtlas/dcp/tests/dataset_runner.py", line 215, in assert_data_browser_bundles
    ).to_return_value(value=True)
  File "/HumanCellAtlas/dcp/tests/wait_for.py", line 27, in to_return_value
    retval = self.func(*self.func_args)
  File "/HumanCellAtlas/dcp/tests/dataset_runner.py", line 222, in _assert_data_browser_bundles
    orange_shortnames = {project["projectShortname"] for hit in response_json["hits"] for project in hit["projects"]}
  File "/HumanCellAtlas/dcp/tests/dataset_runner.py", line 222, in <setcomp>
    orange_shortnames = {project["projectShortname"] for hit in response_json["hits"] for project in hit["projects"]}
TypeError: unhashable type: 'list'
```

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/AZUL-308)
┆Project Name: azul
┆Issue Number: AZUL-308

**Comment by alex-hancock** [comment]
PR WIP [here](https://github.com/HumanCellAtlas/dcp/pull/180)

**Comment by theathorn** [comment]
Please deploy a fix to Integration and Staging a.s.a.p.

**Comment by alex-hancock** [comment]
Accidentally closed this, ignore

**Comment by alex-hancock** [comment]
Closed with [bac2a7c](https://github.com/HumanCellAtlas/dcp/commit/bac2a7cd514ba961a1118a466b1f3b7ded4892e1)
